libkbfs/md_journal.go

```diff
@@ -615,11 +615,36 @@ func (e MDJournalConflictError) Error() string {
 }
 
 // put verifies and stores the given RootMetadata in the journal,
-// modifying it as needed. In particular, if this is an unmerged
-// RootMetadata but the branch ID isn't set, it will be set to the
-// journal's branch ID, which is assumed to be non-zero. As a special
-// case, if the revision of the given RootMetadata matches that of the
-// head, the given RootMetadata will replace the head.
+// modifying it as needed. In particular, there are three cases:
+//
+//  1) rmd is merged. If the journal is empty, then rmd is appended to
```

Comment:
It's weird to say that the rmd is appended to an empty journal (even though that's natural in golang).  Maybe "then rmd becomes the initial entry"?